### ISSUE TITLE
Max 7 compatibility

### ISSUE THREAD

**Issue by botmad** [adrian-gierakowski/ag.granular.suite#1]
Hi Adrian, I installed Max 7 specifically to play with this amazing suite, but I can't get it up & running. Could you confirm whether it works under Max 7 or do I have to revert to Max 6?

**Comment by adrian-gierakowski** [comment]
Max 6 will work. Unfortunately  I haven't tried it with Max 7

**Comment by sone** [comment]
Dear Adrian, are you thinking about updating your granular suite regarding current JamomaMax and MAX7?

**Comment by adrian-gierakowski** [comment]
Unfortunately no. However I would be happy to provide guidance to anyone willing to try :)

**Comment by sone** [comment]
I'm willing to try but I have to find out if I'm able first... Because I've changed the OS nearly everything is more complicated here at my side. JamomaMax is not ready for WIN 64bit and there are no windows developers in the team as Tim mentioned in the google-group. I really would like to help and I'll check out if I can...

**Comment by adrian-gierakowski** [comment]
FTM is another factor. Not sure if it's available for Max7 64bit on
Windows

**Comment by sone** [comment]
Windows FTM 64bit is available for MAX7! It's in the  FTM mailing list shared documents folder.

**Comment by adrian-gierakowski** [comment]
Nice. The bulk of work will be in porting to latest Jamom. The one feature
I think might not be supported by it is the present mixing used in by the
mixpad ( although I might be wrong ). Anyway, what I would do is to start
by posting the engine and player. The you could validate if it works by
recording a sequence in the old version and playing it back in the new one.

**Comment by sone** [comment]
Yes, porting randdist is something I could eventually do, JamomaMax is another story but I have no overview. Tim mentioned in a post a while ago there is no Windows developer in the active project team thus it seems unclear if there will be progress in the near future. So I'll have to do more research to find out what it takes...
